tests/pipeline/test_update_ctx_misc.py

```diff
@@ -33,3 +34,40 @@ def call(self, ctx: Context) -> bool:
     ctx = await pipeline._run_pipeline(Message(), 0, update_ctx_misc={"condition": False})
 
     assert ctx.last_response.text == "failure"
+
+
+@pytest.mark.asyncio
```

Comment:
This mark is not needed since we added [asyncio_mode = "auto"](https://github.com/deeppavlov/chatsky/blob/ff092c5bd07db4b3996f414c3f8b9f3f7ec8de04/pyproject.toml#L219).
Remove here and in `test_update_ctx_misc`.